scripts/synthetic_data/libraries/ije_faker.rb

```diff
@@ -0,0 +1,180 @@
+      "OR" => "Oregon", "CA" => "California", "MA" => "Massachusetts", "PA" => "Pennsylvania", "CO" => "Colorado",
+      "MI" => "Michigan", "RI" => "Rhode Island", "CT" => "Connecticut", "MN" => "Minnesota", "SC" => "South Carolina",
+      "DC" => "District of Columbia", "MS" => "Mississippi", "SD" => "South Dakota", "DE" => "Delaware", "MO" => "Missouri",
+      "TN" => "Tennessee", "FL" => "Florida", "MT" => "Montana", "TX" => "Texas", "GA" => "Georgia", "NE" => "Nebraska",
+      "UT" => "Utah", "HI" => "Hawaii", "NV" => "Nevada", "VT" => "Vermont", "ID" => "Idaho", "NH" => "New Hampshire",
+      "VA" => "Virginia", "IL" => "Illinois", "NJ" => "New Jersey", "WA" => "Washington", "IN" => "Indiana",
+      "NM" => "New Mexico", "WV" => "West Virginia", "IA" => "Iowa", "NY" => "New York", "YC" => "New York",
+      "WI" => "Wisconsin", "KS" => "Kansas", "NC" => "North Carolina", "WY" => "Wyoming", "AS" => "American Samoa",
+      "GU" => "Guam", "MP" => "Northern Marianas", "PR" => "Puerto Rico", "VI" => "Virgin Islands"
+    }
+    @jurisdiction_lookup[jurisdiction]
+  end
+
+  # Generate a certificate number for a jurisdiction and a year; we just start at 1 and increment
+  def self.certificate_number(jurisdiction, year)
+    @certificate_numbers ||= Hash.new { |h, k| h[k] = Hash.new(0) }
+    number = @certificate_numbers[jurisdiction][year] += 1
+  end
+
+  # Generate a marital status based on the age using a distribution from one jurisdiction
+  def self.marital_status(age)
+    # Just set up the weighted distribution once
+    if @marital_distribution.nil?
+      marital_counts = {
+        'M' => 20645,
+        'A' => 228,
+        'W' => 18058,
+        'D' => 10334,
+        'S' => 6286,
+        'U' => 395
+      }
+      @marital_distribution = WeightedDistribution.new(marital_counts)
+    end
+    # Always single if under a somewhat arbitrary age cut off
+    age.to_i > 17 ? @marital_distribution.sample : 'S'
+  end
+
+  # Generate first, middle, and last names adding a suffix to make it clearer that the data is synthetic
+  def self.given_name(sex)
+    name = sex == 'F' ? Faker::Name.female_first_name : Faker::Name.male_first_name
+    "#{name}#{'%02d' % rand(0..99)}"
+  end
+  def self.middle_name
+    name = Faker::Name.middle_name
+    "#{name}#{'%02d' % rand(0..99)}"
+  end
+  def self.last_name
+    name = Faker::Name.last_name
+    "#{name}#{'%02d' % rand(0..99)}"
+  end
+
+  # Load up information on common literal cause of death sequences from a JSON file; this file was generated
+  # using sequences that appear commonly in a set of de-identified jurisdiction data to ensure it's not
+  # connected to any individual decedent so is limited to a relatively small set of possible causes
+  def self.sequence_data
+    @sequence_data ||= JSON.load_file(File.join(File.dirname(__FILE__), '..', 'datafiles', 'common_literal_sequences.json'))
```

Comment:
I keep getting an error on this line when I try to run the script 
<img width="1233" alt="Screenshot 2025-04-14 at 3 39 05 PM" src="https://github.com/user-attachments/assets/7060bfe1-523f-408d-8c49-736cfbd5927a" />
any idea what might be causing it? I ran bundle install successfully

Reply:
after that change, I get a new error saying I have an uninitialized constant
<img width="1283" alt="Screenshot 2025-04-14 at 4 23 02 PM" src="https://github.com/user-attachments/assets/63482d0e-d188-405d-88c6-24cf0cdaeab7" />

Reply:
> after that change, I get a new error

I've pushed an additional small change to try to get things working with older rubies, give it a try